lib/facility_maker.rb

```diff
@@ -0,0 +1,50 @@
+class FacilityMaker
+  attr_reader :add_facility, :facility, :add_facility, :facility_info, :name
+  attr_accessor :name, :address, :phone, :facilities, :add_facility
+
+  def new_facility(state_facilities)
+    state_facilities.map do |state_facility|
+      gen_facility = Facility.new({name: new_facility_name(state_facility), address: new_facility_address(state_facility), 
+      city: state_facility[:city], state: state_facility[:state], zipcode: new_facility_zipcode(state_facility), 
+      phone: new_facility_phone(state_facility)})
+    end
+  end
```

Comment:
This would be a lot easier to read if you added in some line breaks:
```ruby
  def new_facility(state_facilities)
    state_facilities.map do |state_facility|
      Facility.new(
        {
          name: new_facility_name(state_facility),
          address: new_facility_address(state_facility),
          city: state_facility[:city],
          state: state_facility[:state],
          zipcode: new_facility_zipcode(state_facility),
          phone: new_facility_phone(state_facility)
        }
    )
    end
  end
```
Functionally, I changed nothing in your code. All I did was add in some line breaks so I could more easily read and digest what you're trying to do.

Reply:
I like that you created helpers for formatting the data. I might call this method `create_facilities` instead of `new_facility` because it's creating many, not just one but overall you broke this out really nicely.